docs/releases/1.10.txt

```diff
@@ -385,6 +385,15 @@ Dropped support for PostgreSQL 9.1
 Upstream support for PostgreSQL 9.1 ends in September 2016. As a consequence,
 Django 1.10 sets PostgreSQL 9.2 as the minimum version it officially supports.
 
+Changes to the default logging configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``runserver`` command now logs each request to the
+:ref:`django-server-logger` logger instead of outputting it directly to console;
+if your logging config overrides Django's defaults, see the :ref:`Django's
+default logging configuration <default-logging-configuration>` for an example of
```

Comment:
I've added a `.. versionchanged:: 1.10` to `.. versionchanged:: 1.10` pointing to the release notes, and expanded the release note a little bit.